library/network/src/modules/NetworkService.rb

```diff
@@ -238,10 +238,11 @@ def reset!
     end
 
     # Helper to apply a change of the network service
-    def EnableDisableNow
-      return if !Modified()
```

Comment:
I do not know if it is easier to read "return unless force || Modified()"...

Reply:
I personally prefer the way how it is know ... i dislike ```unless``` ;-)